Decision-makers.md

```diff
@@ -0,0 +1,33 @@
+# Decision makers
+
+// This is an experimental document for collecting information about Decision makers important for Ubiquity. Its content and structure will be determined through team discussion.
+
+# Safe 
+
+### Stefan George, Co-Founder and CTO of Gnosis
+
+https://pt.linkedin.com/in/stefangeorge 
+
+https://twitter.com/stefandgeorge
+
+**AMAs with Stefan George**
+
+2022
+https://medium.com/encode-club/stefan-george-co-founder-and-cto-of-gnosis-ama-881266c938e7
+
+2023
+https://www.youtube.com/watch?v=xG9fWAZfGSM
+
+Found by @sergfeldman
```

Comment:
There's a feature called Git blame that shows the history of every line in a document so it might now be necessary to explicitly state that you found the person.